default.nix

```diff
@@ -111,5 +112,23 @@ let
       mainnetWallet = mkDocker { environment = "mainnet"; };
       stagingWallet = mkDocker { environment = "mainnet-staging"; };
     };
+
+    daedalus-bridge = pkgs.runCommand "cardano-daedalus-bridge" {} ''
+      # Generate daedalus-bridge
+      mkdir -p $out/bin $out/config
+      cd $out
+      ${if buildId == null then "" else "echo ${buildId} > build-id"}
+      echo ${gitrev} > commit-id
+
+      cp ${./log-configs}/daedalus.yaml config/log-config-prod.yaml
+      cp ${./lib}/configuration.yaml config
+      cp ${./lib}/*genesis*.json config
```

Comment:
i prefer doing `cp ${./lib + "/configuration.yaml"} config/configuration.yaml` whenever possible, so it doesnt import all of lib into the store
though the wildcards for genesis sort of defeat it in this case

also, my method fails at eval-time if the file is missing, while yours fails at build time, after you have already left because its doing an hour long compile

Reply:
How would I handle the wildcards for genesis without copying `./lib` into the nix store?

Reply:
thats fairly difficult to handle

Reply:
:-) OK I have used the path append method for the log config file but copied the entire `./lib` subdirectory.